include/utils.h

```diff
@@ -141,6 +144,8 @@ void remove_soft_hyphens(std::string& text);
 
 bool is_valid_podcast_type(const std::string& mimetype);
 
+LinkType podcast_mime_to_link_type(const std::string& mimetype, bool& ok);
```

Comment:
I'm tempted to pull in [optional-lite](https://github.com/martinmoene/optional-lite) and rewrite this as `nonstd::optional<LinkType> podcast_mime_to_link_type(const std::string& mimetype)`, but didn't want to add a dependency just because of this. Not sure if we'll be able to avoid it in the long term, though.